x-pack/plugin/security/qa/operator-privileges-tests/src/javaRestTest/java/org/elasticsearch/xpack/security/operator/Constants.java

```diff
@@ -300,6 +303,7 @@
         "cluster:monitor/xpack/searchable_snapshots/stats",
         "cluster:monitor/xpack/security/saml/metadata",
         "cluster:monitor/xpack/spatial/stats",
+        SQL_ASYNC_GET_STATUS_ACTION_NAME,
```

Comment:
I am really curious why we don't use constants for all other actions here. @ywangd or @tvernum could you clarify?

Reply:
The main reason is that the list is generated when the test is first introduced, i.e. the test reports all missing names when this list is empty. Also a few other reasons:
* Many names are not available to this test, e.g. autoscaling, enrich, grok processor, reindex
* Some names are computed, e.g. all the xpack usage actions and shard level actions (`[s]`) and quite a few others
* I find a sorted list of consistent literal strings easier to read and useful to scan through
* Sometimes it is useful to repeat the literal string in tests which can itself act as a test.

Reply:
So, it sounds like we have several good enough reasons to continue using strings here instead of constants, and we should probably continue doing it in this case.

Reply:
Replaced the var with the string (and added the var location as a comment).